assets/src/adaptivity/rules-engine.ts

```diff
@@ -182,6 +183,28 @@ const processRules = (rules: JanusRuleProperties[], env: Environment) => {
       ) {
         modifiedValue = `[${ogValue}]`;
       }
+
+      if (
+        (condition?.type === CapiVariableTypes.ARRAY && condition?.operator === 'containsAnyOf') ||
+        condition?.operator === 'notContainsAnyOf'
+      ) {
+        const targetValue = getValue(condition.fact, env);
+        if (targetValue.charAt(0) !== '[' && targetValue.slice(-1) !== ']') {
+          const modifiedTargetValue = `[${targetValue}]`;
+          const updateAttempt = [
+            {
+              target: `${condition.fact}`,
+              operator: '=',
+              value: modifiedTargetValue,
+            },
+          ];
+          // doing this to avoid updating the same fact multiple times if it exists in multiple rules
+          if (modifiedFacts[condition.fact] === undefined) {
```

Comment:
it shouldn't happen already right? because next time it hits it the targetValue will have the brackets and not enter the condition